### PR TITLE
Add uk-bank-holidays extension

### PR DESCRIPTION
**Pull request by kud**
## Description

Display the upcoming bank holiday information for England and Wales.

## Screencast

Raycast 2024-02-05 at 17 55 59

uk-bank-holidays 2023-10-24 at 18 59 47

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by kud** [comment]
@pernielsentikaer :D

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
What! Sorry @kud - I forgot to hit the submit review button here 😞

**Comment by kud** [comment]
Yeah yeah, did not have time to do it yet :D

File: extensions/uk-bank-holidays/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      "name": "index",
+      "title": "Show Upcoming Bank Holidays in England and Wales",
+      "description": "Show Upcoming Bank Holidays in England and Wales",
+      "mode": "view"
+    },
+    {
+      "name": "next-holiday",
+      "title": "Show Next UK Bank Holiday",
+      "description": "Show Next UK Bank Holiday",
+      "mode": "view"
```
[the remark this answers is not in the document]
Comment (kud) [reply extensions/uk-bank-holidays/package.json:24]: @pernielsentikaer I'm not sure to understand what you actually mean?

**Comment by kud** [comment]
Yeah need to work on this :D

**Comment by kud** [comment]
poke @pernielsentikaer

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by kud** [comment]
🎉

**Pull request merged**